src/Security/Authentication/Negotiate/src/NegotiateHandler.cs

```diff
@@ -57,7 +57,7 @@ protected new NegotiateEvents Events
         /// <returns></returns>
         protected override Task<object> CreateEventsAsync() => Task.FromResult<object>(new NegotiateEvents());
 
-        private bool IsHttp2 => string.Equals("HTTP/2", Request.Protocol, StringComparison.OrdinalIgnoreCase);
+        private bool IsSupportedProtocol => HttpProtocol.IsHttp10(Request.Protocol) || HttpProtocol.IsHttp11(Request.Protocol);
```

Comment:
Stephen was right that Http/1.1 will be more common and should be checked first.